src/gui/core/canvas.cpp

```diff
@@ -462,6 +466,7 @@ text_shape::text_shape(const config& cfg, wfl::action_function_symbol_table& fun
 	, highlight_start_(cfg["highlight_start"])
 	, highlight_end_(cfg["highlight_end"])
 	, highlight_color_(cfg["highlight_color"], color_t::from_hex_string("215380"))
+	, line_spacing_(cfg["line_spacing"].to_int(-1))
```

Comment:
Since this is now a valid key for `[text]` we should look at integrating it into the other label widgets, but we can do that separately.